packages/client/hmi-client/src/page/data-explorer/components/search-results-list.vue

```diff
@@ -7,7 +7,7 @@
 			>
 		</span>
 	</div>
-	<div class="facet-chips">
+	<div v-if="chosenFacets.length > 0" class="facet-chips">
```

Comment:
Is this to remove an empty div?
Just curious because the template below doesnt create anything when length is 0 anyways

Reply:
Yes, it's to remove an empty div. I am trying to add some scroll shadow effects and that empty div was causing me grief.